buildstockbatch/gcp/gcp.py

```diff
@@ -586,10 +587,40 @@ def run_batch(self):
 
         # Start the job!
         created_job = client.create_job(create_request)
+        job_name = created_job.name
+
+        logger.info("Newly created GCP Batch job")
+        logger.info(f"  Job name: {job_name}")
+        logger.info(f"  Job UID: {created_job.uid}")
+        job_url = (
+            "https://console.cloud.google.com/batch/jobsDetail/regions/"
+            f"{self.region}/jobs/{self.unique_job_id}/details?project={self.gcp_project}"
+        )
+        logger.info(f"View GCP Batch job at {job_url}")
 
-        logger.info('Newly created GCP Batch job')
-        logger.info(f'  Job name: {created_job.name}')
-        logger.info(f'  Job UID: {created_job.uid}')
+        # Monitor job status while waiting for the job to complete
+        n_succeeded_last_time = 0
+        client = batch_v1.BatchServiceClient()
+        with tqdm.tqdm(desc="Running Simulations", total=task_count, unit="batch") as progress_bar:
+            job_status = None
+            while job_status not in ("SUCCEEDED", "FAILED", "DELETION_IN_PROGRESS"):
+                time.sleep(10)
```

Comment:
It be nice if sleep time were inversely proportional to percentage done (e.g., 10s @ 0%, 9s @ 10%, ..., 2s at 80%, 1s at 90%), or something like that. 😁

Reply:
Maybe, but the progress here can be pretty batchy, especially if spot instances get preempted. So if one task doesn't get scheduled right away or has to rerun, we might be at 90+% for quite a while.

Either way, that sounds like an improvement to add later. :)